jaxopt/_src/bisection.py

```diff
@@ -63,8 +65,10 @@ class Bisection:
   maxiter: int = 30
   tol: float = 1e-5
   check_bracket: bool = True
-  implicit_diff: bool = True
```

Comment:
Naive question: shouldn't we keep `implicit_diff` as an attribute just in case we want to set it to False?

Reply:
I think unrolling will not work since bisection is just halving the bracketing interval on each iteration. So `implicit_diff=False` doesn't really make sense.

Reply:
I see, also makes sense, thanks! 

Extremely minor, but should `base.IterativeSolver`'s docstring describe `implicit_diff` as a needed attribute then? The only other reason I could find to keep it is what you mentioned in the other comment about kwargs, but probably not worth considering anyway.

Reply:
Done.